dev-docs/docs/codebase/json-schema.mdx

```diff
@@ -0,0 +1,42 @@
+# JSON Schema
+
+```json
+{
+    // Schema information
+    "type": "excalidraw",
```

Comment:
We should also document each attribute in a table format for better clarity first and then attach this example below.

Reply:
Is it something like this
---
<img width="1014" alt="Screenshot 2023-07-28 at 3 11 07 PM" src="https://github.com/excalidraw/excalidraw/assets/82713217/b87f62bd-37a5-4abc-a7e7-b60cf8515ece">